PocketKit/Sources/Sync/RemoteMapping/Collection+remoteMapping.swift

```diff
@@ -0,0 +1,46 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import CoreData
+import PocketGraph
+
+extension Collection {
+    public typealias RemoteCollection = GetCollectionBySlugQuery.Data.Collection
+
+    func update(from remote: RemoteCollection, in space: Space, context: NSManagedObjectContext) {
+        intro = remote.intro
+        publishedAt = remote.publishedAt.flatMap { DateFormatter.clientAPI.date(from: $0) }
+        title = remote.title
+
+        authors = try? NSOrderedSet(array: remote.authors.map {
+            try space.fetchCollectionAuthor(by: $0.name, context: context) ?? CollectionAuthor(context: context, name: $0.name)
+        })
+
+        stories = try? NSOrderedSet(array: remote.stories.enumerated().map {
+            let story = try space.fetchCollectionStory(by: $0.element.url, context: context) ??
+            CollectionStory(
+                context: context,
+                url: $0.element.url,
+                title: $0.element.title,
+                excerpt: $0.element.excerpt,
+                authors: makeStoryAuthors(space: space, context: context, authors: $0.element.authors)
```

Comment:
do we want to add publisher here - maybe the reason why the value is nil (originally did not add it to the init because it is optional on the graph)

Reply:
yep that was the missing bit!